server/service/IndividualServicesService.js

```diff
@@ -469,6 +469,9 @@ exports.startApplicationInGenericRepresentation = function (user, originator, xC
 function getAllApplicationList() {
   return new Promise(async function (resolve, reject) {
     let clientApplicationList = [];
+    let httpClientUuidList = [];
+    let LogicalTerminationPointlist;
+    const FcportValue = 'NewApplicationCausesRequestForInquiringOamRequestApprovals';
```

Comment:
Change the variable "FcportValue" to "forwardingName" as the value indicates a forwardingName.